File: alternatesquares.py
def alternate_sq_sum(arr):
    a = arr
    count = 0
    for i in a:
      if ((a.index(i) % 2) == 0):
        count = count + i
        print i 
        print count
      elif ((a.index(i) % 2) != 0):
        count = count + (i * i)
        print i
        print count
      else:
        print i
    #print count
    return arr
    
    
#alternate_sq_sum([11, 12, 13, 14, 15]) #should return 379
alternate_sq_sum([13, 11, 8, 11, 7, 8, 14, 11, 6, 11, 10, 13, 15, 9, 11, 14])

#1078

File: alternatesquares.js
function alternateSqSum(arr){
  count1 = 0
  count2 = 0
  count = 0
  for (var i = 0; i < arr.length; i++){
    if (arr.indexOf(i) % 2 == 0){
      a = (arr[i] * arr[i])
      count1 = count + a
      console.log(count1)
    }
    else{
      count2 = count + arr[i]
      //console.log(count2)
    }
  }
  console.log(count1)
  return count1 + count2
}

//alternateSqSum([])//,0)
//alternateSqSum([1,2,3,4,5])//,29)
alternateSqSum([-1,0,-3,0,-5,3])//,0)
//alternateSqSum([-1,2,-3,4,-5])//,11)
